data/fields/cuisine.json

```diff
@@ -14,18 +14,26 @@
             "japanese": "Japanese",
             "mexican": "Mexican",
             "kebab": "Kebab",
+            "kebab;pizza": "Kebab and pizza",
```

Comment:
It isn’t necessary to include semicolon-delimited lists, because the `semiCombo` field allows users to mix and match an arbitrary number of these individual values to form a semicolon-delimited list in the raw tag.

Here’s what the field looks like in iD right now. None of the options reflect the strings in this file yet, probably because the latest version of the tagging schema hasn’t been pulled in yet, but at least you can get a sense for how it’ll eventually look:

<img width="378" alt="semiCombo" src="https://user-images.githubusercontent.com/1231218/138037681-443b89b7-1a73-4c43-b280-870ffd753993.png">